From 7beff95c11071170eb27b6fa7d0cc77588caee8e Mon Sep 17 00:00:00 2001
From: Yogesh Tyagi <yogesh.tyagi@intel.com>
Date: Tue, 26 Jul 2022 15:25:10 +0800
Subject: [PATCH] Fix QA Issues

Stop ispc from inserting host file path in generated headers which leads to reproducibility problems.

Upstream-Status: Inappropriate [OE build specific]

Signed-off-by: Yogesh Tyagi <yogesh.tyagi@intel.com>
---
 src/module.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/module.cpp b/src/module.cpp
index e2084d2e..e2626865 100644
--- a/src/module.cpp
+++ b/src/module.cpp
@@ -2555,7 +2555,7 @@ bool Module::writeHeader(const char *fn) {
         perror("fopen");
         return false;
     }
-    fprintf(f, "//\n// %s\n// (Header automatically generated by the ispc compiler.)\n", fn);
+    fprintf(f, "//\n// \n// (Header automatically generated by the ispc compiler.)\n");
     fprintf(f, "// DO NOT EDIT THIS FILE.\n//\n\n");
 
     // Create a nice guard string from the filename, turning any
@@ -2677,7 +2677,7 @@ bool Module::writeDispatchHeader(DispatchHeaderInfo *DHI) {
     FILE *f = DHI->file;
 
     if (DHI->EmitFrontMatter) {
-        fprintf(f, "//\n// %s\n// (Header automatically generated by the ispc compiler.)\n", DHI->fn);
+        fprintf(f, "//\n// \n// (Header automatically generated by the ispc compiler.)\n");
         fprintf(f, "// DO NOT EDIT THIS FILE.\n//\n\n");
     }
     // Create a nice guard string from the filename, turning any
